spec/grape/jwt/grape_usage_spec.rb

```diff
@@ -31,6 +31,49 @@ class API < Grape::API
   end
 end
 
+module TestContentTypeWithPathVersioning
+  class API < Grape::API
+    version 'v1', using: :path
+
+    # formatter :jsonapi, ->() {}
+    content_type :jsonapi, "application/vnd.api+json"
+    default_format :jsonapi
+    format :jsonapi
+
+
+    include Grape::Jwt::Authentication
+    auth :jwt
+
+    resource :test do
+      desc 'A simple GET endpoint which is JWT protected.'
+      post do
+        { test: true }
+      end
+    end
+  end
+end
+
+module TestContentTypeWithoutVersioning
+  class API < Grape::API
+
+    # formatter :jsonapi, ->() {}
+    content_type :jsonapi, "application/vnd.api+json"
+    default_format :jsonapi
+    format :jsonapi
```

Comment:
As I understand the Grape docs, you force the format with `format`. 

```
You can have your API only respond to a single format with format. If you use this, the API will not respond to file extensions other than specified in format. For example, consider the following API.
```

See: https://github.com/ruby-grape/grape#api-formats

---

Could you make another test case here were you allow xml and a custom jsonapi format for instance? (Without the forcing for a single one)